code/modules/wiremod/core/component_printer.dm

```diff
@@ -422,12 +433,15 @@
 	var/list/designs = list()
 
 	var/index = 1
+
 	for (var/list/design as anything in scanned_designs)
 		designs["[index]"] = list(
 			"name" = design["name"],
-			"description" = design["desc"],
-			"materials" = get_material_cost_data(design["materials"]),
-			"categories" = list("Circuitry"),
+			"desc" = design["desc"],
+			"cost" = get_material_cost_data(design["materials"]),
+			"id" = "[index]",
+			"icon" = "integrated_circuit",
+			"categories" = list("/Saved Circuits"),
```

Comment:
/ is intentional?

Reply:
Yes, this is a TG thing, the `/` should be treated like you're defining an absolute path.